prow/config/jobs/metal3-io/project-infra.yaml

```
@@ -239,3 +239,9 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
+  - name: metal3_fullstack_project-infra_building
```

this keyword name is not following conventions for test triggers

```suggestion
  - name: metal3-fullstack-build
```

---

Done.